pixi.toml

```diff
@@ -48,9 +48,10 @@ install = { depends_on = [
 ] }
 
 # format
-check-format = "python .github/common/check_format.py"
+check-format = "echo 'Checking format...'; python .github/common/check_format.py"
```

Comment:
I'm fine with this as is, though I wonder about letting the user pass `-v`/`--verbose` to the format and spelling scripts if they want to see output. Might be situations where we'd prefer no output on success. But the scripts will need slight modification too to accommodate that (they currently show some limited output even on success)